documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -13,14 +14,26 @@ We recommend that you configure clients to use OAUTHBEARER whenever possible.
 OAUTHBEARER provides a higher level of security than PLAIN because client credentials are _never_ shared with Kafka brokers.
 Consider using PLAIN only with Kafka clients that do not support OAUTHBEARER.
 
-If necessary, OAUTHBEARER and PLAIN can be enabled together, on the same `oauth` listener.
+You configure Kafka broker listeners to use OAuth 2.0 authentication for connecting clients.
+If necessary, you can use the OAUTHBEARER and PLAIN mechanisms on the same `oauth` listener.
+The properties to support each mechanism must be explicitly specified in the `oauth` listener configuration. 
 
 .OAUTHBEARER overview
 
-Kafka supports the OAUTHBEARER authentication mechanism, however it must be explicitly configured.
-Also, many Kafka client tools use libraries that provide basic support for OAUTHBEARER at the protocol level.
+OAUTHBEARER is automatically enabled in the `oauth` listener configuration for the Kafka broker.
+You can set the `enableOauthBearer` property to `true`, though this is not required.
 
-To ease application development, Strimzi provides an _OAuth callback handler_ for the upstream Kafka Client Java libraries (but not for other libraries).
+[source,yaml,subs="attributes+"]
+----
+  # ...
+  authentication:
+    type: oauth
+    # ...
+    enableOauthBearer: true
+----
+
+Many Kafka client tools use libraries that provide basic support for OAUTHBEARER at the protocol level.
+To support application development, Strimzi provides an _OAuth callback handler_ for the upstream Kafka Client Java libraries (but not for other libraries).
 Therefore, you do not need to write your own callback handlers for such clients.
```

Comment:
Due to the last part in backets:
> (but not for other libraries)

One might interpret 'such clients' to refer to these other libraries?

```suggestion
Therefore, you do not need to write your own callback handlers.
```